src/applications/mhv/medical-records/util/helpers.js

```diff
@@ -13,6 +13,17 @@ export const dateFormat = (timestamp, format = null) => {
     .format(format || 'MMMM D, YYYY, h:mm a z');
 };
 
+/**
+ * @param {Object} nameObject {first, middle, last, suffix}
+ * @returns {String} formatted timestamp
+ */
+export const nameFormat = ({ first, middle, last, suffix }) => {
+  let name = `${last}, ${first}`;
+  if (middle) name += `, ${middle}`;
```

Comment:
OOC should this have a comma? I would think the usual format would be like `Kennedy, John Fitzgerald` rather than `Kennedy, John, Fitzgerald`

Reply:
Oh, thank you. That was a mistake.